ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>4.8 No tracking commands: How is these tests realised?</p><p> </p><p>I made it to look until here. I close the review for the moment so you can see my comments. </p><p>I will go on later with 4.9 to the end.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
<p>When there is no tracking commands in a period of time, you will see the rotation transition to Fault state and there is the related telemetry (becomes the event in CSC) to check. The Fault state is intentional.</p>